en/views/helpers/form.rst

```diff
@@ -1606,36 +1606,37 @@ You can also change the templates at runtime using the ``templates()`` method::
 List of Templates
 -----------------
 
-A list of the default templates and the variables they can expect are:
-
-* ``button`` {{attrs}}, {{text}}
-* ``checkbox`` {{name}}, {{value}}, {{attrs}}
-* ``checkboxFormGroup`` {{label}}
-* ``checkboxWrapper`` {{label}}
-* ``dateWidget`` {{year}}, {{month}}, {{day}}, {{hour}}, {{minute}}, {{second}}, {{meridian}}
-* ``error`` {{content}}
-* ``errorList`` {{content}}
-* ``errorItem`` {{text}}
-* ``file`` {{name}}, {{attrs}}
-* ``formGroup`` {{label}}, {{input}}, {{error}}
-* ``formStart`` {{attrs}}
-* ``formEnd`` No variables are provided.
-* ``hiddenBlock`` {{content}}
-* ``input`` {{type}}, {{name}}, {{attrs}}
-* ``inputContainer`` {{type}}, {{required}}, {{content}}
-* ``inputContainerError`` {{type}}, {{required}}, {{content}}, {{error}}
-* ``inputSubmit`` {{type}}, {{attrs}}
-* ``label`` {{attrs}}, {{text}}, {{hidden}}, {{input}}
-* ``nestingLabel`` {{hidden}}, {{attrs}}, {{input}}, {{text}}
-* ``legend`` {{text}}
-* ``option`` {{value}}, {{attrs}}, {{text}}
-* ``optgroup`` {{label}}, {{attrs}}, {{content}}
-* ``radio`` {{name}}, {{value}}, {{attrs}}
-* ``radioWrapper``  {{input}}, {{label}}
-* ``select`` {{name}}, {{attrs}}, {{content}}
-* ``selectMultiple`` {{name}}, {{attrs}}, {{content}}
-* ``submitContainer`` {{content}}
-* ``textarea``  {{name}}, {{attrs}}, {{value}}
+A list of the default templates, their default format and the variables they expect are:
+
+* ``button`` : ``<button{{attrs}}>{{text}}</button>``
```

Comment:
Doesn't this make it much easier for the default templates and docs to get out of sync?